mesonbuild/cargo/interpreter.py

```diff
@@ -546,8 +547,13 @@ def _load_manifest(self, subdir: str) -> Manifest:
     def _add_dependency(self, pkg: PackageState, depname: str) -> None:
         if depname in pkg.required_deps:
             return
+        dep = pkg.manifest.dependencies.get(depname)
+        if not dep:
+            if depname in itertools.chain(pkg.manifest.dev_dependencies, pkg.manifest.build_dependencies):
+                # FIXME: Not supported yet
```

Comment:
I think this deserves an mlog.debug() if not an mlog.warning()

Reply:
I agree we should add a warning about build/dev dependencies, but I don't think here is the best place for that. We don't want to warn each time we enable a dev/build dependency, but instead warn once when we have a build.rs we cannot use (done in https://github.com/mesonbuild/meson/pull/13779), or when we have tests defined (not currently supported).